docs/troubleshoot/cli/use-individual-troubleshoot-commands.md

```diff
@@ -10,15 +10,15 @@ Issue the following command:
 zowe -V
 ```
 
-The exact Zowe CLI version may vary depending upon if the `@latest` or `@zowe-v1-lts`, or `@zowe-v2-lts` version is installed.
+The exact Zowe CLI version may vary depending upon if the `@latest`, `@zowe-v1-lts`, `@zowe-v2-lts`, or `@zowe-v3-lts` version is installed.
 
-You can also display the version of your globally-installed Zowe CLI through the following NPM command:
+Display the version of your globally-installed Zowe CLI through the following NPM command:
 
 ```
 npm list -g @zowe/cli
 ```
 
-More information regarding versioning conventions for Zowe CLI and plug-ins is located in [Versioning Guidelines](https://github.com/zowe/zowe-cli/blob/master/docs/MaintainerVersioning.md).
+More information regarding versioning conventions for Zowe CLI and plug-ins is located in [Zowe CLI Releases](https://github.com/zowe/zowe-cli/blob/1ec04a68c7b86acebc26e2af834ce48192794ea0/RELEASE_HISTORY.md).
```

Comment:
This link should be: https://github.com/zowe/zowe-cli/blob/master/RELEASE_HISTORY.md#zowe-v3x-lts-releases

Reply:
i.e., `1ec04a68c7b86acebc26e2af834ce48192794ea0` should be replaced with `master`.